collections/hub/golden_path/feature_tests/duplicate_handling/transfers/fulfill_commit.json

```diff
@@ -112,163 +85,91 @@
-                ]
-              },
-              {
-                "id": 12,
-                "description": "Callback Header - fspiop-source is fspiop-destination in request",
-                "exec": [
-                  "expect(callback.headers['fspiop-source']).to.equal(request.headers['fspiop-destination'])",
-                  ""
-                ]
-              },
-              {
-                "id": 13,
-                "description": "Callback Header - fspiop-desination is fspiop-source in request",
-                "exec": [
-                  "expect(callback.headers['fspiop-destination']).to.equal(request.headers['fspiop-source'])",
-                  ""
-                ]
-              },
-              {
-                "id": 14,
-                "description": "Callback Header - content-type is application/vnd.interoperability.quotes+json;version=1.0",
-                "exec": [
-                  "expect(callback.headers['content-type']).to.equal('application/vnd.interoperability.quotes+json;version={$inputs.expectedQuotesVersion}')",
-                  ""
-                ]
-              },
-              {
-                "id": 15,
-                "description": "Callback Header - accept is empty",
-                "exec": [
-                  "expect(callback.headers['accept']).to.equal(undefined)",
-                  ""
-                ]
-              },
-              {
-                "id": 16,
-                "description": "Callback Header - fspiop-uri is /quotes/quoteId",
-                "exec": [
-                  "expect(callback.headers['fspiop-uri']).to.equal('/quotes/' + request.body.quoteId)",
-                  ""
-                ]
-              },
-              {
-                "id": 17,
-                "description": "Callback Header - fspiop-http-method is PUT",
-                "exec": [
-                  "expect(callback.headers['fspiop-http-method']).to.equal('PUT')",
-                  ""
-                ]
-              },
-              {
-                "id": 18,
-                "description": "Callback Body - quoteId is null",
-                "exec": [
-                  "expect(callback.body.quoteId).to.equal(undefined)",
-                  ""
-                ]
-              },
-              {
-                "id": 19,
-                "description": "Callback Body - ilpPacket is not null",
-                "exec": [
-                  "expect(callback.body.ilpPacket).to.not.equal(undefined)",
-                  ""
-                ]
-              },
+              }
+            ]
+          },
+          "ignoreCallbacks": true,
+          "scripts": {
+            "preRequest": {
+              "exec": [
+                ""
+              ]
+            },
+            "postRequest": {
+              "exec": [
+                ""
+              ]
+            }
+          }
+        },
+        {
+          "id": 7,
+          "description": "Send Quote - Response",
```

Comment:
How do we know its ok to send the Quote Response here?

I don't see any await inboundEvent listener in the `POST /Quotes` to determine it was received by the NORESPONSE_SIMPAYEE_NAME.

Otherwise we may have a race condition with the `PUT /Quotes` request below.